ui/package.json

```diff
@@ -45,7 +45,7 @@
     "rxjs": "^6.6.6",
     "superagent": "^7.1.3",
     "timezones-list": "3.0.1",
-    "tsx": "^3.4.0",
+    "tsx": "^3.4.2",
```

Comment:
I think this should be moved to devDependencies.

Reply:
Let’s the bot know?

Reply:
Sorry, not sure I understand the question.

Reply:
Oh lol gotcha. Was just commenting for anyone tempted to test/merge this.  :-P

But agreed, Snyk should be funnier. 0/10 on comedy.